CONTRIBUTING.md

```diff
@@ -171,6 +171,10 @@ The list of labels for current issues can be found [here][turing-way-labels] and
 
 - [![outreach](https://img.shields.io/badge/-Outreach-fcbae8.svg)][labels-outreach] _These issues relate to topics to reach out to the community._
 
+- [![good-first-PR-review](https://img.shields.io/badge/-good%20first%20PR%20review-C992E0.svg)](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20PR%20reviews) _These pull requests are for the new maintainers of The Turing Way to start with reviewing and merging tasks_
+
+  If you are a new maintainer of The Turing way and are anxious about how to start reviewing and merging various contributions, these pull requests act as a great starting point for you to begin. Issues like small modifications, typo errors and minor bug fixes are resolved by these PRs which are easy to review as a beginner.
```

Comment:
```suggestion
  If you are a new member of The Turing way and are looking for opportunities to start as a reviewer of contributions made on our Github repository, these pull requests are a great starting point for you. Issues like small modifications, typo errors and minor bug fixes are resolved by these PRs which are easy to review as a beginner.
```

Reply:
Looks better! I will try to improve and become more clear in future. Thank you!